ogr/ogrsf_frmts/sqlite/ogrsqlitevirtualogr.cpp

```diff
@@ -43,19 +43,48 @@
 #include "cpl_conv.h"
 #include "cpl_error.h"
 #include "cpl_string.h"
-#include "gdal_priv.h"
-#include "ogr_api.h"
-#include "ogr_core.h"
-#include "ogr_feature.h"
-#include "ogr_geometry.h"
-#include "ogr_p.h"
-#include "ogr_spatialref.h"
-#include "ogrsf_frmts.h"
-#include "ogrsqlite3ext.h"
-#include "ogrsqlitesqlfunctions.h"
-#include "ogrsqliteutility.h"
-#include "ogr_swq.h"
-#include "sqlite3.h"
+
+/************************************************************************/
+/*                  OGR2SQLITE_GetNameForGeometryColumn()               */
+/************************************************************************/
+
+CPLString OGR2SQLITE_GetNameForGeometryColumn(OGRLayer *poLayer)
+{
+    const char *pszGeomColumn = poLayer->GetGeometryColumn();
+    if (pszGeomColumn != nullptr && !EQUAL(pszGeomColumn, ""))
+    {
+        if (poLayer->GetLayerDefn()->GetFieldIndex(pszGeomColumn) < 0)
+            return pszGeomColumn;
+    }
+
+    CPLString osGeomCol("GEOMETRY");
+    int bTry = 2;
```

Comment:
```suggestion
    int iTry = 2;
```

isn't `b` prefix for booleans?

Reply:
fixed